### PR TITLE
feat(payment): STRIPE-99 Add flag to validate unmount

### PR DESCRIPTION
**Pull request by EliLV21**
## What? [STRIPE-99]()
Needs to add a flag to do a validation

## Why?
Needs to add a flag to validate if the error show it is correct

## Testing / Proof

Screen Shot 2022-09-20 at 17 32 40

@bigcommerce/checkout @bigcommerce/payments

### CODE REVIEW

**Review by AlanMunozN — APPROVED** [review]
LGTM

**Comment by BCHMercado** [comment]
💚  testing done
https://bigcommercecloud.atlassian.net/jira/software/c/projects/STRIPE/boards/622?modal=detail&selectedIssue=STRIPE-99

File: packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
     private _stripeElements?: StripeElements;
     private _isMounted = false;
     private _unsubscribe?: (() => void);
+    private _isDesinitialize?: boolean;
```
Comment (animesh1987) [inline packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts:37]:
```suggestion
    private _isDeinitialize?: boolean;
```

same for the other places

Comment (EliLV21) [reply packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts]: Hi Animesh, I alredy do the changes, can you do the validation again, please?

**Review by animesh1987 — APPROVED** [review]

**Pull request merged**